src/pools/fixedRate/FixedRateExchange.ts

```diff
@@ -120,6 +120,8 @@ export class FixedRateExchange {
    * @param {String} account
    * @param {String} dtAmount datatoken amount we want to buy
    * @param {String} datatokenAddress datatokenAddress
+   * @param {String} consumeMarketAddress consumeMarketAddress
```

Comment:
it would really help developers if this here and wherever we use fees as params, has a short sentence about the fees. Who gets it, when does it happen, what unit to pass. See https://github.com/oceanprotocol/market/blob/v4/app.config.js#L25-L43 for some copy/paste